crates/e2e/tests/e2e/buffers.rs

```diff
@@ -128,7 +128,11 @@ async fn onchain_settlement_without_liquidity(web3: Web3) {
         .await
         .unwrap();
     // Check that internal buffers were used
-    assert!(settlement_contract_balance == U256::ZERO);
+    assert!(
```

Comment:
If you want to the error to print the actual balance you can also just use `assert_eq` and compare to zero.